model-api/xml-model/src/main/java/org/camunda/bpm/model/xml/impl/validation/ModelValidationResultsImpl.java

```diff
@@ -75,9 +74,50 @@ public void write(StringWriter writer, ValidationResultFormatter formatter) {
     }
   }
 
+  @Override
+  public void write(StringWriter writer, ValidationResultFormatter formatter, int maxSize) {
```

Comment:
🔧 Most Java writers use `offset` and `len` parameters to control what is written. I think that `len` is too strict for this use case, as it requires knwoing beforehand exactly how many bytes/chars to write. In that sense, limiting the writer to some maximum number of bytes/chars makes more sense. My feeling is that this is what you mean by `maxSize`, but size seems unfitting for writing. I suggest changing `maxSize` to `limit`.

Reply:
Thanks Nico!
I initially had named it `limit` actually, and then changed it to `maxSize`. As the javadoc also implies, this param is meant to have the `the maximum size (in bytes) that is allowed to be written to the writer`. And I personally thought `limit` would be confusing to describe that. Because if I personally read limit, I could think that it's a limit for the number of errors or such.